From fe537a2063afb6142050865b7f23c0674e941b13 Mon Sep 17 00:00:00 2001
From: dhead666 <myfoolishgames@gmail.com>
Date: Wed, 5 Mar 2014 20:21:37 +0200
Subject: [PATCH] libhts patch

---
 lib/libhts/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/libhts/Makefile.am b/lib/libhts/Makefile.am
index 002dbcf..e2cb320 100644
--- a/lib/libhts/Makefile.am
+++ b/lib/libhts/Makefile.am
@@ -8,6 +8,7 @@ libhts_la_SOURCES = htsmsg.c \
                     sha1.c
 
 libhts_la_CPPFLAGS=-std=c99 -D_GNU_SOURCE
+libhts_la_LDFLAGS=-lgcc
 
 $(LIB): libhts.la
 	cp -f .libs/libhts.a .
-- 
1.9.0

